students/kpersic/module_3/blog/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Dog blog">
+    <title>Dog blog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;700&display=swap" rel="stylesheet">
+    <script src="https://use.fontawesome.com/48f1032c77.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+
+        <header class="header">
+            <h1>Welcome to my blog</h1>
+            <p class="greeting">Hello world, this is my blog about things!</p>
+        </header>
+
+        <main>
+
+            <section class="section_wrapper">
```

Comment:
Nisam siguran koje se konvencije ovdje držiš, ali preporučam BEM: http://getbem.com/

Nekako se najčešće vrijednosti class atributa pišu u kebab- ili CamelCase notaciji (CamelCase zna biti kada se radi s Reactom i sličnim component-based frameworcima). Dakle ovo bi vrlo vjerojatno onda bilo `section-wrapper`.

BEM ti može pomoći razmišljati o klasama i strukturiranju komponenti na malo drugačiji način. Tu se još često primjenjuje i ideja iza nekakvog "objektno-orijentiranog CSS-a" (to ne znači ono što misliš), a to je "separate container and content" i "separate structure and skin". Ponekad se različiti sustavi organiziranja CSS-a kolju pa se naravno ne očekuje da ih miješaš i koristiš sve, ali dobar dio ideja koje promiču su apsolutno valjane i treba ih imati na umu kada se pišu stilovu. O tome možeš više pročitati ovdje: https://github.com/stubbornella/oocss/wiki